src/client/components/measures-list/measures-list.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="measures-list">
     <template v-if="measures">
-      <div
+      <section
```

Comment:
Do not agree that this is a section. The headers are part of the list and not part of the document outline.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

"Do not use the section element as a generic container; this is what <div> is for, especially when the sectioning is only for styling purposes. A rule of thumb is that a section should logically appear in the outline of a document."